### DISCUSSION TITLE
I would like to help and i saw that most discussions are unanswered. If its ok I could help mostly handle answer the discussions and issues

### DISCUSSION THREAD

**Discussion by Tedd-Codes** [minbrowser/min#2470] [General]
I would like to help and i saw that most discussions are unanswered. If its ok I could help mostly handle answer the discussions and issues. Also help add features.

**Comment by PalmerAL** [comment]
Yeah sure, help is always appreciated. Although, most of the discussions don't really have an "answer", but are really feature requests/ideas.

I don't want to sound too critical, but looking at #2453 - it's better to slow down and give a thorough answer, rather than answering as many posts as possible :) Like the actual work to be done there is figuring out what the best way to implement that would be, is there a library we can use, a specification...

**Reply by Tedd-Codes** [reply]
thx
